ledger-core/virtual/lmn/register.go

```diff
@@ -8,23 +8,15 @@
 package lmn
```

Comment:
package name can be confusing - may be "vnlmn"?

Reply:
fixed